wsclient/src/main/java/org/hpccsystems/ws/client/utils/Utils.java

```diff
@@ -64,14 +68,24 @@ public static String parseVersionFromWSDLURL(String wsdlurl)
 
     static public void println(PrintStream stream, String message, boolean onlyifverbose, boolean verbosemode )
     {
-        if (verbosemode || !onlyifverbose)
-            stream.println(message);
+        // if (verbosemode || !onlyifverbose)
```

Comment:
let's label these 2 methods deprecated. Let's add a new log method log (String message, level(error|debug|trace|etc.) to be more in tune w/ log4j